contracts/truefi2/LoanFactory2.sol

```diff
@@ -37,13 +35,12 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
     // @dev Track Valid LoanTokens
     mapping(ILoanToken2 => bool) public override isLoanToken;
 
-    IPoolFactory public poolFactory;
+    address private DEPRECATED__poolFactory;
     address private DEPRECATED__lender;
     address public liquidator;
 
     address public admin;
 
-    ICreditModel public creditModel;
```

Comment:
Thanks for being vigilant! Though I don't think this would cause storage corruption because we haven't deployed all the changes yet. Take a look at https://etherscan.io/address/0x84aa7f36a397cad693e70224930d925b1e98e085#code#F20#L30.